ci-operator/config/rh-ecosystem-edge/kernel-module-management/rh-ecosystem-edge-kernel-module-management-main.yaml

```diff
@@ -0,0 +1,32 @@
+binary_build_commands: make manager
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+images:
```

Comment:
I don't see here our image build. It will be the equivalent of https://github.com/openshift/release/blob/d6441930099066f6012457f2a4cb2912d77c9fa7/ci-operator/config/openshift/special-resource-operator/openshift-special-resource-operator-master.yaml#L16-L21.

We should probably not replace the base image for the m/s repo though.